nginx/config.go

```diff
@@ -145,6 +145,12 @@ http {
 			return 200;
 		}
 
+    {{ if $routerConfig.statusEndpoint }}
+    location /nginx_status {
+      stub_status on;
+    }
+    {{ end }}
+
```

Comment:
Yes. We've already done that very same thing here: https://github.com/deis/router/blob/master/nginx/config.go#L202-L207

This makes me wonder a few things.

- Are you aware that lots of router metrics are already available in Grafana? (Just making sure.)
- Are the metrics that are already exposed at `127.0.0.1:9090/stats` something you can work with? i.e. Is this new endpoint needed for something you can't already get?
- Purely to satisfy my own curiosity, what agent / metric collector would you be running that would be hitting this from the router's own loopback interface?